go/vt/vttablet/tabletserver/throttle/mysql/probe.go

```diff
@@ -14,45 +49,35 @@ import (
 
 // Probe is the minimal configuration required to connect to a MySQL server
 type Probe struct {
-	Key             InstanceKey
+	Alias           string
 	MetricQuery     string
 	Tablet          *topodatapb.Tablet
-	TabletHost      string
-	TabletPort      int
 	CacheMillis     int
 	QueryInProgress int64
 }
 
-// Probes maps instances to probe(s)
-type Probes map[InstanceKey](*Probe)
+// Probes maps tablet aliases to probe(s)
```

Comment:
Yeah it's a good point. It will be yet another somewhat significant change what with all the tests. I'm happy to give this a go in a followup PR, as this PR is already bloated as it is.